internal/v1/api.yaml

```diff
@@ -2121,6 +2121,23 @@ components:
           type: integer
           description: Group id of the group to create (optional)
     User:
+      oneOf:
+        - $ref: '#/components/schemas/UserPassword'
+        - $ref: '#/components/schemas/UserKey'
```

Comment:
This may be done better ?  I feel like with two schemas I introduced more logic which is needed in code to work with User (always interpret is as a user with either key or password). 
Would you prefer to just have one schema for user with ssh key and password and somewhere in code check that at least one is non empty? (which is basically what the schema should validate :sweat_smile: ). Since I don't know the best practice here I am just questioning what I did.

Reply:
also having both is valid, which you can't do through one of.

Reply:
Now the validation is done in `CreateBlueprint`, schema requires only name for each User, but the custom validation needs either password or ssh_key